src/lib/parse-component-email.test.js

```diff
@@ -0,0 +1,19 @@
+const {parseComponentEmail} = require('./parse-component-email.js')
```

Comment:
any way you can use `import {parseComponentEmail} from './parse-component-email.js'` 
You can do that via Babel `babel-preset-es2015` to be able to interpret ES6
with `.babelrc` like this:
```
{
    "presets": ["es2015"]
}
```

Reply:
Can you say more about how to do this? I change to use `import` and added the `.babelrc` but `yarn test` still fails with syntax error. What other configuration is required? I'll need the tests to run and deployment as the Lambda function to work.

Reply:
It is something we can do later if you want to merge this now